app/models/network/battery.rb

```diff
@@ -7,5 +7,9 @@ module Network
   # inherit from the Storage class.
   class Battery < Storage
     extend ProfileScaled
+
+    def self.disabled?(options)
+      options[:battery_storage] == 'false'
```

Comment:
I'm not keen on `false` being a string. I'm guessing this is coming from the `params` hash, but the Rails-side of the application should be casting these values to `true` or `false` before passing them into the Network module.

The Network and Market namespaces will likely be extracted out of Moses and into a separate Gem, so it's a good practice to already treat it like an external library.